src/allmydata/web/common.py

```diff
@@ -51,6 +68,7 @@
     IResource,
 )
 
+from allmydata.dirnode import ONLY_FILES, _OnlyFiles
```

Comment:
It's a bit annoying that this private name needs to be imported here just for the type annotation.  But the function being annotated has an awful type so maybe the implementation should be annoying.  Not actionable, I guess.  I'll just add `parse_replace_arg` to my imaginary list of Tahoe functions to fix...